app/services/proofing/aamva/response/verification_response.rb

```diff
@@ -96,6 +101,10 @@ def parse_response
         def rexml_document
           @rexml_document ||= REXML::Document.new(http_response.body)
         end
+
+        def mva_timeout?(error_message)
+          error_message.include? MVA_TIMEOUT_EXCEPTION
```

Comment:
style super nit, I would use parens here
```suggestion
          error_message.include?(MVA_TIMEOUT_EXCEPTION)
```

the rule of thumb I've used in the past is statement vs expression, since this is an expression (we use the result of it), we use the parens. Statements don't need parens